mu/logic.py

```diff
@@ -413,7 +463,12 @@ def flash(self):
             logger.debug('Flashing to device.')
             hex_file = os.path.join(path_to_microbit, 'micropython.hex')
             uflash.save_hex(micropython_hex, hex_file)
+            rt_hex_path = get_runtime_hex_path()
             message = 'Flashing "{}" onto the micro:bit.'.format(tab.label)
+            if (rt_hex_path != "" and os.path.exists(rt_hex_path)):
```

Comment:
That's just to determine whether to display the path in the popup dialog when flashing. We only want to show the path if it was actually used (it might point to a path which doesn't exist, in which case the default runtime will have been used).